intel-sgx/enclave-runner/src/loader.rs

```diff
@@ -332,21 +342,25 @@ impl<'a> EnclaveBuilder<'a> {
     }
 
     pub fn build<T: Load>(mut self, loader: &mut T) -> Result<Command, anyhow::Error> {
+        let num_worker_threads = self.num_worker_threads.unwrap_or_else(num_cpus::get);
+        anyhow::ensure!(num_worker_threads > 0, "`num_worker_threads` cannot be zero");
+        anyhow::ensure!(num_worker_threads <= 512, "`num_worker_threads` cannot exceed 512");
```

Comment:
Is this needed at all? It might be unlikely but I don't think we should _prevent_ people from spawning more threads than there are cores. It's just inefficient, but not wrong.

Reply:
Right. That's a good point. But maybe we should enforce a reasonable maximum? It doesn't have to be 512, but maybe we should somehow prevent specifying something like `1 << 60` as `num_worker_threads`.